<launch>
  <param name="debug_mode" type="bool" value="false"/>
  <param name="joint_size" type="int" value="8"/>
  <param name="max_angular_speed" type="double" value="0.51419526535"/> <!-- PI/10 rad/s -->
  <param name="update_rate" type="double" value="1000.0"/>

  <group ns="ur5">
    <node pkg="ur5" type="vision-node.py" name="vision_node" output="screen"/>
    <!-- <node pkg="ur5" type="pointcloud_node.py" name="pointcloud_node" output="screen"/> -->
    <node pkg="ur5" type="desired_poses_node.py" name="desired_poses_node" output="screen"/>
    <!-- <node pkg="ur5" type="desired_poses_params.py" name="desired_poses_params" output="screen"/> -->

    <node pkg="ur5" type="task_planner" name="task_planner" output="screen"/>
    <!-- <node pkg="ur5" type="robot_controller" name="robot_controller" output="screen"/>
    <node pkg="ur5" type="motion_planner" name="motion_planner" output="screen"/> -->
  </group>
</launch>
